pandas/io/formats/style.py

```diff
@@ -341,9 +341,16 @@ def set_tooltips(
         >>> df.style.set_tooltips(ttips, css_class='tt-add', props=[
         ...     ('visibility', 'hidden'),
         ...     ('position', 'absolute'),
-        ...     ('z-index', 1)])
+        ...     ('z-index', 1)])  # doctest: +SKIP
+            0	1
+        0	0	1
+        1	2	3
```

Comment:
I dont this this output will display aligned (could be wrong), but it also isn't really necessary to display here. The `set_tooltips` function amends the class object and manipulates the HTML result. This output is just a distractor I think.